docs/reference/function-configuration/function-configuration-reference.md

```diff
@@ -94,9 +94,9 @@ The `spec` section contains the requirements and attributes and has the followin
 | readinessTimeoutSeconds | int | Number of seconds that the controller will wait for the function to become ready before declaring failure (default: 60) |
 | avatar | string | Base64 representation of an icon to be shown in UI for the function |
 | eventTimeout | string | Global event timeout, in the format supported for the `Duration` parameter of the [`time.ParseDuration`](https://golang.org/pkg/time/#ParseDuration) Go function |
-| securityContext.runAsUser | int | The UID to run the entrypoint of the container process. (k8s only) |
-| securityContext.runAsGroup | int | The GID to run the entrypoint of the container process. (k8s only) |
-| securityContext.fsGroup | int | A supplemental group added to the groups to run the entrypoint of the container process. (k8s only) |
+| securityContext.runAsUser | int | The user ID (UID) for runing the entry point of the container process; applicable only to Kubernetes platforms |
```

Comment:
Actually, @quaark added support for that, it is applicable on docker platform as well.
all the three of `runAsUser`, `runAsGroup` and `fsGroup`

Reply:
I've now edited the Kubernetes-only comment for all three `securityContext` fields to refer also to Docker — commit 37b424e.

Reply:
UPDATE: As agreed offline, I've now removed the platform-specific notes from all `securityContext` fields — commit d48e67a.